external/fv3fit/tests/test_models.py

```diff
@@ -66,11 +67,11 @@ def test_nonnegative_model_outputs():
     model = DenseModel("sample", ["input"], ["output"], hyperparameters,)
     batch = xr.Dataset(
         {
-            "input": (["sample"], np.arange(100)),
+            "input": (["x"], np.arange(100)),
             # even with negative targets, trained model should be nonnegative
-            "output": (["sample"], np.full((100,), -1e4)),
+            "output": (["x"], np.full((100,), -1e4)),
         }
     )
     model.fit([batch])
-    prediction = model.predict(batch)
+    prediction = model.predict(stack_non_vertical(batch))
```

Comment:
You should be able to revert the changes in this test and it should still pass. Alternatively, shouldn't `predict` take in an un-stacked dataset at least in one test?